File: Day9/removeDuplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) == 0 or nums == None: return None
        i = 1
        ptr = nums[0]
        
        while i < len(nums):
            if nums[i] == ptr:
                nums.pop(i)
            else:
                ptr = nums[i]
                i+=1

        return len(nums)

File: Day9/removeDuplicates.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() == 0) return 0;
        
        int ptr = nums.at(0);
        int i = 1;
        
        while (i < nums.size()) {
            
            if (nums.at(i) == ptr) {
                nums.erase(nums.begin()+i);
                }
            
            else{
                ptr = nums.at(i);
                i += 1;
            }
    }
    return nums.size();
        
    }
};
